playbooks/app_install/roles/docker_install/vars/main.yml

```diff
@@ -1,15 +1,35 @@
 ---
 # vars file for docker_install
-# Which docker version should be installed
-docker_version: '23.0.6'
+docker_version: 24.0.6
```

Comment:
Ah, didn't see the version already being provided here.  Could we use "docker_version" in docker_install/tasks/main.yml file instead of hard-coding there?